x-pack/legacy/plugins/apm/public/components/app/Settings/CustomizeUI/CustomLink/LicensePrompt.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiButton, EuiEmptyPrompt } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+import { useKibanaUrl } from '../../../../../hooks/useKibanaUrl';
+
+export function LicensePrompt() {
+  const licensePageUrl = useKibanaUrl(
+    '/app/kibana',
+    '/management/elasticsearch/license_management/home'
+  );
+  return (
+    <EuiEmptyPrompt
+      iconType="iInCircle"
+      iconColor=""
+      title={
+        <h2>
+          {i18n.translate(
+            'xpack.apm.settings.customizeUI.customLink.license.title',
+            {
+              defaultMessage: 'Start free 14-day trial'
+            }
+          )}
+        </h2>
+      }
+      body={
+        <>
+          <p>
+            {i18n.translate(
+              'xpack.apm.settings.customizeUI.customLink.license.text',
+              {
+                defaultMessage:
+                  "To create custom links, you must be subscribed to an Elastic Gold license or above. With it, you'll have the ability to create custom links to improve your workflow when analyzing your services."
```

Comment:
We already have a very similar component for Service Maps. Would it make sense to re-use the same component for both? Perhaps with an option to supply a custom message (like this one)?

Reply:
https://github.com/elastic/kibana/blob/master/x-pack/legacy/plugins/apm/public/components/app/ServiceMap/PlatinumLicensePrompt.tsx#L47

Reply:
Sure it would. 👍🏻

Reply:
Btw I' not sure why the custom link says 14-days trial and service maps is 30 days. We should verify which is correct.

Reply:
@formgeist can you tell which one should be the correct one? 🤔

Reply:
refactored on 1cfd8be

Reply:
@cauemarcondes The trial is 30 days (https://www.elastic.co/guide/en/elasticsearch/reference/current/start-trial.html) - that's a typo on my part.